pkg/tasks/tasks.go

```diff
@@ -351,6 +352,28 @@ func WithUpgrade(t Tasks, followers ...kubeoneapi.HostConfig) Tasks {
 		append(
 			Task{Fn: restartKubeAPIServer, Operation: "restarting unhealthy kube-apiserver"},
 			Task{Fn: upgradeStaticWorkers, Operation: "upgrading static worker nodes"},
+			Task{
+				Fn:          migratePVCAllocatedResourceStatus,
+				Operation:   "migrating PVCs",
+				Description: "migrating PVCs",
+				Predicate: func(s *state.State) bool {
+					targetVersion, err := semver.NewVersion(s.Cluster.Versions.Kubernetes)
+					if err != nil {
+						// This should never ever happen because we validate the version.
+						panic(err)
```

Comment:
I know I just drilled panics into you, but this here might just be a t.Fatal(). But six of the ones, half a dozen of the others.